src/unix/linux_like/linux/gnu/b64/x86_64/mod.rs

```diff
@@ -285,7 +285,6 @@ s_no_extra_traits! {
         pub uc_mcontext: mcontext_t,
         pub uc_sigmask: ::sigset_t,
         __private: [u8; 512],
-        __ssp: [::c_ulonglong; 4],
```

Comment:
Instead of removing the field, can you comment it out, and add a comment about why it is commented out ? For example:

> // FIXME: the shadow stack field requires glibc >= 2.28. 
> // Re-add once we drop compatibility with glibc versions older than 2.28.